packages/elements/list/src/insertFragmentList.ts

```diff
@@ -31,6 +40,21 @@ export const insertFragmentList = (editor: PlateEditor) => {
     return [Node.get(root, ancestor), ancestor];
   };
 
+  const findListItemsWithContent = <T extends TDescendant>(first: T): T[] => {
```

Comment:
This method is added to handle a case when a sublist is selected from a different level, so it will normalize it and move all the LIs to the same level. e.g. let's say from this list

```
- a
  - b
    - c
    - d
  - e
    - f
```

the following part is selected:

```
    - c
    - d
  - e
    - f
```

With this method the LIs `c` and `d` will be at the same level as `e` so that will be properly inserted at the right level.